From e94097d9912c893ff84110f2f77d9c19c6582c10 Mon Sep 17 00:00:00 2001
From: bhatnagarshasya <shasya.bhatnagar20@st.niituniversity.in>
Date: Mon, 27 Feb 2023 22:54:23 +0530
Subject: [PATCH] bhatnagarshasya's commit

---
 scratch.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/scratch.py b/scratch.py
index dba0b39..1efac3e 100644
--- a/scratch.py
+++ b/scratch.py
@@ -1,5 +1,7 @@
 def add(a,b):
     return a+b
+def div(a,b):
+    return a/b
 def calculator(a,b,op):
     ans=0
     if op==1:
-- 
2.36.0.windows.1

